.github/workflows/semconvgen.yml

```
@@ -68,3 +68,11 @@ jobs:
         else
           tag_and_push "${GITHUB_REF#"refs/tags/"}"
         fi
+    - name: Push the Dev Docker image
+      if: startsWith(github.ref, 'refs/pull/243')
+      run: |
+        echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u "${{ secrets.DOCKER_USERNAME }}" --password-stdin
+        function tag_and_push {
+          docker tag semconvgen "otel/semconvgen:${1}" && docker push "otel/semconvgen:${1}"
+        }
+        tag_and_push "0.24.0-dev-pr-243"
```

```suggestion
        TAG="${GITHUB_REF#"refs/heads/"}"
        TAG="${TAG/"/"/"-"}"
        tag_and_push "${TAG}"
```
This should result in a label `feature-codegen-by-namespace` as Docker image labels don't allow slashes.

---

thank you!